src/tests.rs

```diff
@@ -291,6 +291,87 @@ mod tests {
             Some("operation not supported on this platform".to_string())
         );
     }
+
+    #[pg_test]
+    #[search_path(@extschema@)]
+    #[should_panic]
+    fn pgx_can_panic() {
+        panic!()
+    }
+
+    #[pg_test]
+    #[search_path(@extschema@)]
+    #[should_panic]
+    fn plrust_can_panic() {
+        let definition = r#"
+            CREATE FUNCTION shut_up_and_explode()
+            RETURNS text AS
+            $$
+                panic!();
+                None
+            $$ LANGUAGE plrust;
+        "#;
+
+        Spi::run(definition);
+        let retval = Spi::get_one::<String>("SELECT shut_up_and_explode();\n");
+        assert_eq!(retval, None);
+    }
+
+    #[pg_test]
+    #[search_path(@extschema@)]
+    #[should_panic]
+    fn postgrestd_subprocesses_panic() {
+        let definition = r#"
+            CREATE FUNCTION say_hello()
+            RETURNS text AS
+            $$
+                let out = std::process::Command::new("echo")
+                    .arg("Hello world")
+                    .stdout(std::process::Stdio::piped())
+                    .output()
+                    .expect("Failed to execute command");
+                Some(String::from_utf8_lossy(&out.stdout).to_string())
+            $$ LANGUAGE plrust;
+        "#;
+        Spi::run(definition);
+
+        let retval = Spi::get_one::<String>("SELECT say_hello();\n");
+        assert_eq!(retval, Some("Hello world\n".into()));
+    }
+
+    /// This test is... meta. It's intended to sleep enough to usually go last.
+    /// If a previous test panics in a way so severe it aborts Postgres?
+    /// This test will fail as a result.
+    /// Ideally this should be slow but still sometimes finish second-to-last.
```

Comment:
Out of curiosity, what happens if this test does finish second-to-last?

Reply:
Nothing, but if it never finishes not-last, it's too slow (my initial draft had it add about a minute to the full test suite on my machine, a test suite which only takes about 2 minutes locally).

Reply:
Ah ok, thanks!